ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -522,6 +524,15 @@ object Config {
             "Experimental buffer for Ledger API streaming queries. Should not be used in production."
           )
           .action((_, config) => config.copy(enableInMemoryFanOutForLedgerApi = true))
+
+        // TODO ha: remove after stable
+        opt[Unit]("index-ha-unsafe")
```

Comment:
```suggestion
        opt[Unit]("experimental-index-ha")
```
We don't want customers reading `unsafe` in our help messages ;-)

Reply:
I was merely following the convention with the other feature flags we have currently for the indexer. Like the suggestion though, did it!